platform/android/export/export.cpp

```diff
@@ -43,6 +43,7 @@
 #include "editor/editor_log.h"
 #include "editor/editor_node.h"
 #include "editor/editor_settings.h"
+#include "platform/android/export/gradle_export_util.h"
```

Comment:
Is it expected that this header is used in this PR, but not in #40182?

Reply:
It will be used in #40182. The difference is that (up until about 15 minutes ago), the 3.2 branch did not have the `gradle_export_util.h` file